blocks/cards/cards.css

```diff
@@ -66,7 +66,10 @@
   margin-top: var(--spacing-large);
 }
 
-.cards.profile-summary .cards-card-body h1,h2,h3,h4 {
+.cards.profile-summary .cards-card-body h1,
+h2,
+h3,
+h4 {
   font-size: var(--font-size-1100);
```

Comment:
Rules for h2, h3, h4 are open ended and should be covered by context of component
.cards.profile-summary .cards-card-body h1,
.cards.profile-summary .cards-card-body h2,
.cards.profile-summary .cards-card-body h3,
.cards.profile-summary .cards-card-body h4

Reply:
okay